gef-ui/server/controllers/application-submission/index.js

```diff
@@ -28,7 +28,7 @@ const postApplicationSubmission = async (req, res, next) => {
   const { comment } = body;
   const application = await Application.findById(dealId, user, userToken);
   const { submissionType } = application;
-  const maker = await api.getUserDetails(application.userId, userToken);
+  const currentStatus = application.status;
```

Comment:
Great stuff, can we remove the mock `getUserDetails` calls and variables in the test as well since it's not being used now?

Reply:
Done